data/fixtures/scopes/python/collectionItem.unenclosed.iteration3.scope

```diff
@@ -2,10 +2,25 @@ def foo():
     global bar, baz
 ---
 
-[Range] = 1:11-1:19
+[#1 Range] =
+[#1 Domain] = 0:8-0:8
+          ><
+0| def foo():
+
+
+[#2 Range] = 1:4-1:19
+      >---------------<
```

Comment:
This syntax is indicating where Cursorless might _search_ for items (Andreas says: that's the definition of iteration scope), although not necessarily were they could be returned. The >/< are not part of the range.